tests/Feature/Models/GoalTest.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace Tests\Feature\Models;
+
+use App\Models\Goal;
+use App\Models\User;
+use Carbon\Carbon;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Tests\TestCase;
+
+/** @group goal */
+class GoalTest extends TestCase
+{
+    use RefreshDatabase;
+
+    /** @test */
+    public function goal_get_achieved_attribute_false()
+    {
+        $goal = Goal::factory()->make([
+            'balance' => 0.5,
+            'amount' => 1
```

Comment:
Please add trailing commas to the end of every array item (below too) to match the php-cs-fixer rules

Reply:
+